src/en/aurora/src/eu/kanade/tachiyomi/extension/en/aurora/Aurora.kt

```diff
@@ -0,0 +1,205 @@
+package eu.kanade.tachiyomi.extension.en.aurora
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.Request
+import okhttp3.Response
+import rx.Observable
+import java.lang.Exception
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+/**
+ *  @author THE_ORONCO <the_oronco@posteo.net>
+ */
+
+class Aurora : HttpSource() {
+
+    override val name = "Aurora"
+    override val baseUrl = "https://comicaurora.com"
+    override val lang = "en"
+    override val supportsLatest = false
+    private val authorName = "OSP-Red"
+    private val auroraGenre = "fantasy"
+    private val dateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale.US)
+
+    override fun chapterListRequest(manga: SManga): Request = throw Exception("Not used")
+
+    override fun chapterListParse(response: Response): List<SChapter> = throw Exception("Not used")
+
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        return Observable.just(fetchChapterListTR(baseUrl + manga.url, mutableListOf()))
+    }
+
+    private tailrec fun fetchChapterListTR(
+        currentUrl: String,
+        foundChapters: MutableList<SChapter>
+    ): MutableList<SChapter> {
+        val currentPage = client.newCall(GET(currentUrl, headers)).execute().asJsoup()
+
+        val pagesAsChapters = currentPage.select(".post-content")
+            .map { postContent ->
+                val chapterUrl = postContent.select("a.webcomic-link").attr("href")
+                val title = postContent.select(".post-title a").text()
+                val chapterNr = title.substringAfter('.').toFloat()
+                val dateString = postContent.select(".post-date").text()
+                val date = dateFormat.parse(dateString)?.time ?: 0L
+
+                SChapter.create().apply {
+                    url = chapterUrl
```

Comment:
Not sure if I missed this one or you did. :thinking: 
```suggestion
                    setUrlWithoutDomain(chapterUrl)
```